org.hydrogenmusic.hydrogen.json

```diff
@@ -0,0 +1,177 @@
+      "name": "jack1",
+      "config-opts": [
+        "--libdir=/app/lib"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://jackaudio.org/downloads/jack-audio-connection-kit-0.125.0.tar.gz",
+          "sha256": "3517b5bff82139a76b2b66fe2fd9a3b34b6e594c184f95a988524c575b11d444"
+        }
+      ],
+    },
+    {
+      "name": "lash",
+      "config-opts": [
+        "--disable-static",
+        "--disable-serv-inst",
+        "CFLAGS=-D_GNU_SOURCE"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.savannah.gnu.org/releases/lash/lash-0.5.4.tar.gz",
+          "sha256": "105a7da84415c4725c6bcad28e70f23aeb4534f94fc80ca262b6a2cef2226c16"
+        },
+        {
+          "type": "patch",
+          "strip-components": 0,
+          "path": "patches/lash-0.5.3-no-static-lib.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/lash-gcc47.patch"
+        },
+        {
+          "type": "patch",
+          "path": "patches/lash-linking.patch"
+        }
+      ]
+    },
+    {
+      "name": "portaudio",
+      "config-opts": [
+        "--disable-static",
+        "--without-oss"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz",
+          "sha256": "f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513"
+        }
+      ]
+    },
+    {
+      "name": "portmidi",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=/app/lib",
+        "-DCMAKE_ARCHIVE_OUTPUT_DIRECTORY=/app/lib",
+        "-DCMAKE_RUNTIME_OUTPUT_DIRECTORY=/app/bin"
+      ],
+      "post-install": [
+      /* MuseScore looks for this name for some reason */
+        "ln -s libportmidi.so /app/lib/libporttime.so"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/portmedia/files/portmidi/217/portmidi-src-217.zip",
+          "sha256": "08e9a892bd80bdb1115213fb72dc29a7bf2ff108b378180586aa65f3cfd42e0f"
+        },
+        {
+          "type": "patch",
+          "path": "patches/portmidi-no-java.patch"
+        }
+      ],
+      "cleanup": [
+        "/bin",
+        "/lib/pkgconfig",
+        "/include",
+        "*.a",
+        "*.la"
+      ]
+    },
+    {
+      "name": "liblo",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://download.sf.net/sourceforge/liblo/liblo-0.29.tar.gz",
+          "sha256": "ace1b4e234091425c150261d1ca7070cece48ee3c228a5612d048116d864c06a"
+        }
+      ]
+    },
+    {
+      "name": "hydrogen",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DWANT_JACK=0",
+        "-DWANT_PORTAUDIO=1",
+        "-DWANT_PORTMIDI=1"
+      ],
+      "build-options": {
+        "env": [
+          "LADSPA_PATH=/app/lib/ladspa"
```

Comment:
Don't you have to set this in `finish-args` too, to make the app find the files at runtime? At least, that's what i had to do for some apps I maintain.